engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -2309,6 +2370,24 @@ static void LogFrameBufferError(GLenum status)
         };
 
         uint8_t max_color_attachments = PFN_glDrawBuffers != 0x0 ? MAX_BUFFER_COLOR_ATTACHMENTS : 1;
+        bool check_attachments_same_size = false;
+
+        // Emscripten: WebGL 1 requires the "WEBGL_draw_buffers" extension to load the PFN_glDrawBuffers pointer,
+        //             but for WebGL 2 we have support natively and no way of loading the pointer via glfwGetprocAddress
+    #if __EMSCRIPTEN__
+        if (context->m_MultiTargetRenderingSupport)
+        {
+            max_color_attachments = MAX_BUFFER_COLOR_ATTACHMENTS;
+            check_attachments_same_size = true;
+
+            if (!ValidateFramebufferAttachments(max_color_attachments, buffer_type_flags, color_buffer_flags, creation_params))
+            {
+                dmLogError("All attachments must have the same size!");
+                return 0;
+            }
```

Comment:
This is a webgl caveat only, which is super annoying.. Maybe we should state that in the docs